src/Uno.UWP/Devices/Radios/Radio.Android.cs

```diff
@@ -0,0 +1,252 @@
+#if __ANDROID__

```

Comment:
#nullable enable

Reply:
It leads to many errors that I don't know how to resolve. Is this really neccessary?

Reply:
Windows.Devices.Radios.Radio has no constructor, so how to resolve: `Non-nullable property 'Name' must contain a non-null value when exiting constructor. Consider declaring the property as nullable.`
It cannot be marked nullable, as this is part of UWP  API.